lms/envs/common.py

```diff
@@ -370,6 +370,9 @@
 
     # Set this to False to facilitate cleaning up invalid xml from your modulestore.
     'ENABLE_XBLOCK_XML_VALIDATION': True,
+
+    # Set this to the TPA provider_id if you want the entire site to be behind TPA
+    'FORCED_TPA_PROVIDER_ID': '',
```

Comment:
I think you still need a `configuration` PR to actually pipe this setting into the `FEATURES` dict.